PowerEditor/src/MISC/PluginsManager/PluginsManager.cpp

```diff
@@ -470,10 +470,10 @@ void PluginsManager::notify(const SCNotification *notification)
 			}
 			catch (...)
 			{
-				TCHAR funcInfo[128];
+				TCHAR funcInfo[256];
 				generic_sprintf(funcInfo, TEXT("notify(SCNotification *notification) : \r notification->nmhdr.code == %d\r notification->nmhdr.hwndFrom == %p\r notification->nmhdr.idFrom == %d"),\
 					scNotif.nmhdr.code, scNotif.nmhdr.hwndFrom, scNotif.nmhdr.idFrom);
-				pluginCrashAlert(_pluginsCommands[i]._pluginName.c_str(), funcInfo);
+				pluginCrashAlert(_pluginInfos[i]->_moduleName.c_str(), funcInfo);

```

Comment:
That's the correct fix. However, there are still some function use `_pluginsCommands[i]._pluginName` instead `_pluginInfos[i]->_moduleName` (not all, but some). Could you fix them as well?